src/client/Microsoft.Identity.Client/Internal/Logger/LoggerHelper.cs

```diff
@@ -110,6 +113,8 @@ internal static string GetPiiScrubbedExceptionDetails(Exception ex)
                 {
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "HTTP StatusCode {0}", msalServiceException.StatusCode));
                     sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "CorrelationId {0}", msalServiceException.CorrelationId));
+                    var aadError = JsonHelper.GetExistingOrEmptyString(JsonHelper.ParseIntoJsonObject(msalServiceException.ResponseBody), OAuth2ResponseBaseClaim.ErrorDescription).Split(':').FirstOrDefault();
```

Comment:
We shouldn't do parsing here but on `MsalServiceException` where extensive parsing already takes place. Btw there already exists "SubError" and "ErrorCodes" which may be what you want.

Reply:
Ah, yes I can use ErrorCodes